### PR TITLE
US : Change United Nations Web TV link

### PR DESCRIPTION
**Pull request by BellezaEmporium**
Audio issues are seen, but are server-related.

Also bumps quality to 1080p.

### CODE REVIEW

**Review by what-a — APPROVED** [review]

**Comment by what-a** [comment]
Audio issues on the new stream are indeed severe, but the old stream is dead. So, something is better than nothing. 😁

**Comment by Sphinxroot** [comment]
https://www.youtube.com/channel/UC5O114-PQNYkurlTg6hekZw/live

File: streams/us.m3u [hunk 1]
```diff
@@ -1747,7 +1747,7 @@ https://na-all23.secdn.net/pegstream3-live/play/65ea794b-dd82-41ce-8e98-a9177289
 #EXTINF:-1 tvg-id="UCTVUniversityofCalifornia.us" status="online",UCTV University of California (720p) [Not 24/7]
 https://59e8e1c60a2b2.streamlock.net/509/509.stream/playlist.m3u8
 #EXTINF:-1 tvg-id="UNWebTV.us" status="error",UN Web TV (540p)
-https://bcliveunivsecure-lh.akamaihd.net/i/un150_A1_1@575439/master.m3u8
+https://cdnapi.kaltura.com/p/2503451/sp/250345100/playManifest/entryId/1_gb6tjmle/protocol/https/format/applehttp/a.m3u8
```
Comment (freearhey) [inline streams/us.m3u:1750]: Are you sure this is a UN Web TV stream? 

Here's what I see:
image

and that's not what the channel's website is showing
image

Comment (LaneSh4d0w) [reply streams/us.m3u:1750]: The link I gave is from the United Nations website, and it's written "24hr stream"

Comment (LaneSh4d0w) [reply streams/us.m3u:1750]: United Nations meetings stream : https://cdnapi.kaltura.com/p/2503451/sp/250345100/playManifest/entryId/1_4ghd8fth/protocol/https/format/applehttp/a.m3u8

**Review by MapGuy11 — APPROVED** [review]

**Pull request merged**